src/WSEvents.cpp

```diff
@@ -1453,6 +1457,44 @@ void WSEvents::OnSceneItemVisibilityChanged(void* param, calldata_t* data) {
 	instance->broadcastUpdate("SceneItemVisibilityChanged", fields);
 }
 
+/**
+ * An item's locked status has been toggled.
+ *
+ * @return {String} `scene-name` Name of the scene.
+ * @return {String} `item-name` Name of the item in the scene.
+ * @return {int} `item-id` Scene item ID
+ * @return {boolean} `item-locked` New locked state of the item.
+ *
+ * @api events
+ * @name SceneItemLockedChanged
+ * @category sources
+ * @since unreleased
+ */
+void WSEvents::OnSceneItemLockedChanged(void* param, calldata_t* data) {
+	auto instance = reinterpret_cast<WSEvents*>(param);
+
+	obs_scene_t* scene = nullptr;
+	calldata_get_ptr(data, "scene", &scene);
```

Comment:
You can use the `calldata_get_pointer` helper instead, which directly returns the pointer returned by the callback.